client/components/jetpack/daily-backup-status/status-card/backup-successful.jsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * Internal dependencies
+ */
+import { useTranslate } from 'i18n-calypso';
+import { applySiteOffset } from 'lib/site/timezone';
+import { useLocalizedMoment } from 'components/localized-moment';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import getSiteTimezoneValue from 'state/selectors/get-site-timezone-value';
+import getSiteGmtOffset from 'state/selectors/get-site-gmt-offset';
+import getRewindCapabilities from 'state/selectors/get-rewind-capabilities';
+import ActivityCard from 'components/activity-card';
+import ActionButtons from '../action-buttons';
+import BackupChanges from '../backup-changes';
+import useGetDisplayDate from '../use-get-display-date';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+import cloudSuccessIcon from './icons/cloud-success.svg';
+
+const BackupSuccessful = ( { backup, deltas, selectedDate } ) => {
+	const translate = useTranslate();
+	const siteId = useSelector( getSelectedSiteId );
+	const hasRealtimeBackups = useSelector( ( state ) =>
+		getRewindCapabilities( state, siteId )?.includes( 'backup-realtime' )
+	);
+
+	const moment = useLocalizedMoment();
+	const timezone = useSelector( ( state ) => getSiteTimezoneValue( state, siteId ) );
+	const gmtOffset = useSelector( ( state ) => getSiteGmtOffset( state, siteId ) );
+
+	const getDisplayDate = useGetDisplayDate();
+	const displayDate = getDisplayDate( backup.activityTs );
+	const displayDateNoLatest = getDisplayDate( backup.activityTs, false );
+
+	const today = applySiteOffset( moment(), {
+		timezone: timezone,
+		gmtOffset: gmtOffset,
+	} );
+	const isToday = selectedDate.isSame( today, 'day' );
+
+	const meta = get( backup, 'activityDescription[2].children[0]', '' );
+
+	// We should only showing the summarized ActivityCard for Real-time sites when the latest backup is not a full backup
+	const showBackupDetails =
+		hasRealtimeBackups &&
+		( 'rewind__backup_complete_full' !== backup.activityName ||
+			'rewind__backup_only_complete_full' !== backup.activityName );
+
+	return (
+		<>
+			<div className="status-card__message-head">
+				<img src={ cloudSuccessIcon } alt="" role="presentation" />
+				<div className="status-card__hide-mobile">
+					{ isToday ? translate( 'Latest backup' ) : translate( 'Latest backup on this day' ) }
+				</div>
+			</div>
+			<div className="status-card__hide-desktop">
+				<div className="status-card__title">{ displayDate }</div>
+			</div>
+			<div className="status-card__hide-mobile">
+				<div className="status-card__title">{ displayDateNoLatest }</div>
+			</div>
+			<div className="status-card__meta">{ meta }</div>
+			<ActionButtons rewindId={ backup.rewindId } />
+			{ showBackupDetails && (
+				<div className="status-card__realtime-details">
+					<div className="status-card__realtime-details-card">
+						<ActivityCard activity={ backup } summarize />
+					</div>
+				</div>
+			) }
+			{ ! hasRealtimeBackups && <BackupChanges deltas={ deltas } /* metaDiff={ metaDiff */ /> }
```

Comment:
I suggest removing the lingering comment if we don't need it :)

Reply:
I debated removing it, but I wasn't sure if this was a feature we'd be adding back at some point in the future. It was removed before because of an issue with non-English locales, if I recall correctly. Here's an issue link I found on the main Backup page: `1169345694087188-as-1176670093007897`